objectivec/GPBWellKnownTypes.m

```diff
@@ -115,3 +115,40 @@ - (void)setTimeIntervalSince1970:(NSTimeInterval)timeIntervalSince1970 {
 }
 
 @end
+
+NSString *const GPBTypeGoogleApisComPrefix = @"type.googleapis.com/";
+
+@implementation GPBAny (GBPWellKnownTypes)
+
+- (NSString*)getAnyTypeName {
```

Comment:
This should be a property declared in the interface.

@property (readonly) NSString *typeName;

the implementation should be
- (NSString *)typeName {
  return [self.typeURL subStringFromIndex:[GPBTypeGoogleApisComPrefix length]];
}